tests/PYME/localization/test_localization_algorithms.py

```diff
@@ -40,6 +40,26 @@ def test_LatGaussFitFR():
     
     assert errors_over_pred_IQR < 2.5
 
+# def test_Dumbell3DFitR():
```

Comment:
maybe we can remove this?

Reply:
I think Zach is being nice and leaving it commented out as it currently doesn't pass / he hasn't found the right text criteria and doesn't want to add a failing test.

Reply:
This was my thought, yes. I do believe this test will pass in the near future.